docs/secpoll.zone

```diff
@@ -1,4 +1,4 @@
-@       86400   IN  SOA pdns-public-ns1.powerdns.com. peter\.van\.dijk.powerdns.com. 2023100501 10800 3600 604800 10800
+@       86400   IN  SOA pdns-public-ns1.powerdns.com. peter\.van\.dijk.powerdns.com. 2023101103 10800 3600 604800 10800
```

Comment:
I think this could be `01`, not `03`, but I don't care much :)

Reply:
Right, I did not check and emacs helpfully auto-updated it for me :shrug: